Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,524 @@
+
+### Sister projects
+
+The sister projects are led independently. They synchronize with the CVA6 main and sub-projects as deemed necessary.
+
+At the time of writing, the sister projects are:
+- [CVA6-H](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/CORE-V-CVA6-H) (CVA6 Hypervisor Extension Support)
+- [CORE-V L1 DCACHE](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/CORE-V-L1-DCACHE) (High Performance Data Cache)
+- [CORE-V-VEC](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/CORE-V-VEC-Research)  (vector coprocessor)
+- [CORE-V CV-Mesh](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/CV-MESH) (multi-core coherence system)
+- [Tightly-coupled cache coherence for CVA6](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/CVA6_tightly_coupled_cache_coherence)
+- [CVA6 Free-RTOS](https://github.com/openhwgroup/programs/blob/master/Project-Descriptions-and-Plans/FreeRTOS/PA%20for%20CVA6%20FreeRTOS%20support.md)
+- [Superscalar CVA6](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/cva6-dual-issue)
+
+## Project Leader(s)
+
+For the main project:
+- Project Manager (PM) and Technical project leader (TPL): Jérôme Quévremont, TRT
+- Verification leader and main committer: Jean-Roch Coulon, TSS
+- FPGA softcore leader: Sébastien Jacq, TRT
+
+The sub-projects will name their leadership teams.
+
+## Project Planning Documents
+
+The main project workplan is below.
+
+## Summary of requirements
+
+### CVA6 features
+
+The CVA6 specification was prepared for the 2022-02-28 PA gate and approved by the Technical WG. It is available in [ReadTheDocs](https://docs.openhwgroup.org/projects/cva6-user-manual/02_cva6_requirements/cva6_requirements_specification.html).
+
+To avoid duplications and inconsistencies, it is not copied here.
+
+### License scheme
+
+Because of the nature of its business, Thales expectation is to contribute and get a **sustainable open-source solution**
+- to integrate CVA6 in new ASIC and FPGA projects for a long period and to maintain/upgrade them for several decades for industrial domains (avionic, satellite, railways, energy...);
+- to permit audit and reviews of verification and tools in the context of certifiable security and functional safety;
+- to permit public review to improve the quality of the core and its ecosystem;
+- to foster cooperative projects and connections with academy and research.
+
+Thereforce, Thales expects:
+- Not only the core RTL, but also a version of the verification environment and the SW tools are available as open-source.
+- Commercial tools can be added to this basic set to deliver additional value (improved quality, coverage...).
+    - But the project shall not depend on a single-source commercial tool.
+	- Commercial tools with multiple sources, such as logic simulators, are acceptable.
+
```

Comment:
Fair to describe this goal. Not sure if "License" is the area.

Reply:
I'll change the title. Licenses are addressed elsewhere.